centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx

```diff
@@ -447,7 +453,7 @@ describe('Edit Panel', () => {
       cy.findByRole('checkbox').should('not.be.checked').and('be.disabled');
     });
 
-    cy.get('#panel-content').scrollTo('bottom');
+    cy.get('#panel-content').scrollTo('center');
```

Comment:
why not to bottom ?
snapshots are totally differents

Reply:
@kduret When Bottom is applied the area of "notification channels" are not visible in the viewport ^^

Reply:
Whereas the test case talks about notification channels ^^

Reply:
then you should use [scrollIntoView](https://docs.cypress.io/api/commands/scrollIntoView) to scroll to the notification section

Reply:
@kduret done !